src/main/kotlin/no/nav/syfo/domain/inntektsmelding/Inntektsmelding.kt

```diff
@@ -75,4 +76,9 @@ data class Inntektsmelding(
             )
         )
     }
+
```

Comment:
Tidligere var denne logikken i Consumer (og ga beng i vedtaksperiodeID) - vanskelig å teste consumer, så la logikken her for å gjøre det lettere å teste.. Men er litt grisete.. Skal forhåpentligvis kunne slettes igjen, dersom vi alltid har en vedtaksperiodeId sammen med en selvbestemt IM - og det tror jeg vi kan kreve.